common/djangoapps/edraak_i18n/middleware.py

```diff
@@ -15,6 +15,8 @@ class ForceLangMiddleware(object):
     """
     def process_request(self, request):
         if 'HTTP_ACCEPT_LANGUAGE' in request.META:
+            old_language = request.META['HTTP_ACCEPT_LANGUAGE']
+            request.META['REQUEST_LANGUAGE'] = old_language
```

Comment:
The idea is great. May I suggest another name:
 - `ORIGINAL_HTTP_ACCEPT_LANGUAGE `